content/static/js/custom.cfr.js

```diff
@@ -17,4 +20,64 @@ function checkNavbar() {
   } else {
     showNavbar();
   }
-};
\ No newline at end of file
+};
+
+var dummy, dummy2, dummyJSON, txt, x, xmlhttp;
+
+function submitCFR() {
+  dummy = { name1: document.getElementById("cfrName").value, num1: document.getElementById("cfrNum1").value,
+  num2: document.getElementById("cfrNum2").value, num3: document.getElementById("cfrNum3").value};
+  dummyJSON = JSON.stringify(dummy);
+  localStorage.setItem("dummyJSONtest", dummyJSON);
+  xmlhttp = new XMLHttpRequest();
+
+  if (testing == "true"){
```

Comment:
[NON-ESSENTIAL] If this is information is really just for testing, it would be better print to the Javascript console instead of inserting elements into the page. However, maybe we want to include a "Added row" on the page after hitting submit, in which case this shouldn't depend on the 'testing' variable and it should be moved to execute after the POST request returns successfully.

Reply:
This code is adding a row to the table, but I was under the impression that in the final version we will be using a GET request to access the data to populate the table and for that reason I didn't just have it executing after the POST request.